cvmfs-config-osg.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=daemon to keep /cvmfs/config-osg.opensciencegrid.org mounted
+After=autofs.service
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/cvmfs-config-osgd -p /run/cvmfs-config-osgd.pid /cvmfs/config-osg.opensciencegrid.org
+PIDFile=/run/cvmfs-config-osgd.pid
```

Comment:
>    I think this should have a Requires statement for autofs. After only designates only the sequencing, not the dependency.

Added.  It does not replace the After functionality, so need both.

>    Might also be possible to put a requirement on the /cvmfs mount itself.

I don't see how.

>    Automatic restart?

I don't know what you mean by that.

>    Is the PID file actually necessary for a service of type simple?

Oh, you're right.  I assumed it was needed so systemd would be able to kill it, but it must keep track of the PID separately.

> Can we make it so we can restart this service whenever autofs is restarted?

Oh, maybe that's what you meant by "Automatic restart?"  Yes that would be good.  As a matter of fact, the "Requires" and "After" already make this service restart with "systemctl restart autofs".  They do not make it start after "systemctl stop autofs" (that stops cvmfs-config-osg) followed by "systemctl start autofs" (that does not start cvmfs-config-osg).  For that, I believe a "WantedBy=autofs.service" in the [Install] section would normally work, but it doesn't, I think because there is not actually an autofs.service file in the autofs package (it is just inferred by /etc/init.d/autofs).  It does however work to do "ln -s ../cvmfs-config-osg.service /lib/systemd/system/autofs.service.wants/cvmfs-config-osg.service", I will add that to the package.